Mathlib/Analysis/Complex/ValueDistribution/CharacteristicFunction.lean

```diff
@@ -62,4 +62,58 @@ lemma characteristic_sub_characteristic_eq_proximity_sub_proximity (h : Meromorp
     characteristic f ⊤ - characteristic (f · - a₀) ⊤ = proximity f ⊤ - proximity (f · - a₀) ⊤ := by
   simp [← Pi.sub_def, characteristic, logCounting_sub_const h]
 
+/-!
+## Behaviour under Arithmetic Operations
+-/
+
+/--
+For `1 ≤ r`, the characteristic function of `f * g` at zero is less than or
+equal to the sum of the characteristic functions of `f` and `g`, respectively.
+-/
+theorem characteristic_zero_mul_le {f₁ f₂ : ℂ → ℂ} {r : ℝ} (hr : 1 ≤ r)
+    (h₁f₁ : MeromorphicOn f₁ Set.univ) (h₂f₁ : ∀ z ∈ univ, meromorphicOrderAt f₁ z ≠ ⊤)
+    (h₁f₂ : MeromorphicOn f₂ Set.univ) (h₂f₂ : ∀ z ∈ univ, meromorphicOrderAt f₂ z ≠ ⊤) :
```

Comment:
1. I see we have no `Meromorphic f := MeromorphicOn f Set.univ` predicate. Can you please open a Zulip thread about adding one? I won't say it's necessary for this PR, but we have these sorts of predicates for everything else throughout the library (e.g., `Continuous`, `ContinuousOn`, `ContinuousAt`).
2. You should be omitting `∈ univ` (and I suspect this is also a symptom partially caused by (1)).

```suggestion
    (h₁f₁ : MeromorphicOn f₁ Set.univ) (h₂f₁ : ∀ z, meromorphicOrderAt f₁ z ≠ ⊤)
    (h₁f₂ : MeromorphicOn f₂ Set.univ) (h₂f₂ : ∀ z, meromorphicOrderAt f₂ z ≠ ⊤) :
```
This will require fixing the proof ever so slightly.